torch/utils/_benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -0,0 +1,326 @@
+import collections
+import dataclasses
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+from typing import List, Tuple
+
+import torch
```

Comment:
Since you're writing this file from scratch, why not challenge yourself to make it typecheck entirely? :)

Reply:
Done.